### PR TITLE
Jormungandr prometheus api

### PR DESCRIPTION
**Pull request by const-iohk**
[VIT-1422](https://jira.iohk.io/browse/VIT-1422)

Removing jormungandr-monitor and using jormungandr /prometheus api endpoint instead.

### CODE REVIEW

File: jobs/tasks/telegraf.cue [hunk 1]
```diff
@@ -40,7 +39,7 @@ import (
 		[inputs.prometheus]
 		metric_version = 1
 
-		urls = [ "http://{{ env "NOMAD_ADDR_\(#prometheusPort)" }}" ]
+		urls = [ "http://{{ env "NOMAD_ADDR_rest)/prometheus" }}" ]
```
Comment (manveru) [inline jobs/tasks/telegraf.cue]: This should be:

    urls = [ "http://{{ env "NOMAD_ADDR_rest" }}/prometheus" ]

**Review by manveru — CHANGES_REQUESTED** [review]
Seems mostly good, apart from the missing flake.lock and wrong consul-template syntax.

File: jobs/tasks/telegraf.cue (hunk 1 above)
Comment (const-iohk) [reply jobs/tasks/telegraf.cue]: yes, my bad while editing, thanks!

File: jobs/tasks/telegraf.cue [hunk 2]
```diff
@@ -40,7 +39,7 @@ import (
 		[inputs.prometheus]
 		metric_version = 1
 
-		urls = [ "http://{{ env "NOMAD_ADDR_\(#prometheusPort)" }}" ]
+                urls = [ "http://{{ env "NOMAD_ADDR_rest" }}/prometheus" ]
```
Comment (manveru) [inline jobs/tasks/telegraf.cue]: This needs to be indented with tabs like the others.

**Review by manveru — APPROVED** [review]
LGTM

File: jobs/tasks/telegraf.cue (hunk 2 above)
Comment (const-iohk) [reply jobs/tasks/telegraf.cue]: Yes, saw the error, handled the whitespace correctly now. Thanks!

**Comment by garbas** [comment]
deprecated by #44

**Pull request closed without merging**